src/Android/Xamarin.Android/Samples/Security/OAuth/readme.metadata.json

```diff
@@ -19,9 +23,12 @@
         "/net/latest/android/sample-code/oauth.htm"
     ],
     "relevant_apis": [
-        "ArcGISPortal",
-        "AuthenticationManager",
-        "Credential"
+        "",
```

Comment:
This looks like an issue - might need to be fixed on both branches

Reply:
Was due to an errant space in the readmes. I've fixed in this branch and will do a separate PR for master